components/bootloader/src/main/esp32.bootloader.ld

```diff
@@ -21,7 +21,7 @@ MEMORY
 }
 
 /*  Default entry point:  */
-ENTRY(call_start_cpu0);
+ENTRY(boot_loader_entry);
```

Comment:
Although I agree it's a bit odd, the call_start_cpu0 function name is a a convention in two places (here and the app entry point) and it's documented in the description of the boot flow in "doc/general-notes.rst".) Would prefer keeping the existing name here.